src/Native/System.Net.Http.Native/CMakeLists.txt

```diff
@@ -1,10 +1,9 @@
 project(System.Net.Http.Native)
 
-find_library(CURL NAMES curl)
-if(CURL STREQUAL CURL-NOTFOUND)
-    message(SEND_ERROR "!!! Cannot find libcurl and System.Net.Http.Native cannot build without it. Try installing libcurl4-openssl-dev (or the appropriate package for your platform) !!!")
-    return()
-endif()
+find_package(CURL)
+if(NOT CURL_FOUND)
+    message(FATAL_ERROR "!!! Cannot find libcurl and System.Net.Http.Native cannot build without it. Try installing libcurl4-openssl-dev (or the appropriate package for your platform) !!!")
+endif(NOT CURL_FOUND)
 
```

Comment:
The curl issue is separate from the rest of the PR, right? Could you split the commits? You can keep the 1 PR.

(Disregard earlier deleted comment about it not being in the description, I just missed it.)